mmv1/products/compute/Interconnect.yaml

```diff
@@ -381,7 +381,7 @@ properties:
       description: |
         interconnects.list of features requested for this Interconnect connection
       values:
-        - :MACSEC
+        - :IF_MACSEC
```

Comment:
👍 added that option back in with a note. I've kept it as an enum for now because the server-side error described in b/362553823 doesn't seem to give obvious feedback that a bad option was used.